azure-functions/create-function-app-connect-to-document-db/create-function-app-connect-to-documentdb.sh

```diff
@@ -0,0 +1,25 @@
+# login to azure 
+az login
```

Comment:
`az login` is done as a step in the wrapper and doesn't need to be captured here.